clearwater-infrastructure/usr/share/clearwater/infrastructure/scripts/config_validation/project_clearwater_options.py

```diff
@@ -0,0 +1,154 @@
+#
+# @file project_clearwater_options.py
+#
+# Copyright (C) Metaswitch Networks 2017
+# If license terms are provided to you in a COPYING file in the root directory
+# of the source code repository by which you are accessing this code, then
+# the license outlined in that COPYING file applies to your use.
+# Otherwise no rights are granted except for those provided to you by
+# Metaswitch Networks in a separate written agreement.
+
+import os
+
+import cw_infrastructure.validators as vlds
+from cw_infrastructure.check_config_utilities import (OK, ERROR, error,
+                                                      number_present, Option)
+
+
+def get_options():
+    """Set up the list of options to be validated"""
+    options = [
+        Option('local_ip', Option.MANDATORY, vlds.ip_addr_validator),
```

Comment:
What's defined inclusion on this list?

Reply:
Everything that previous check config scripts have checked. Anything else that's easy to check. It's not meant to be a full set, but it's meant to be sufficient.